var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -23,6 +23,7 @@ class Magma(CMakePackage, CudaPackage, ROCmPackage):
     test_requires_compiler = True
 
     version('master', branch='master')
+    version('2.6.2', sha256='ff49ee0b07ab20205667b5af08615b80406c05a2a9a5b5e4c65bcfee8759a6d0')
     version('2.6.2rc1', commit='5959b8783e45f1809812ed96ae762f38ee701972')
     version('2.6.1', sha256='6cd83808c6e8bc7a44028e05112b3ab4e579bcc73202ed14733f66661127e213', preferred=True)
```

Comment:
We can remove `preferred=True` and deprecate the rc version now

Reply:
Done with [a02a5ff](https://github.com/spack/spack/pull/29531/commits/a02a5fff8d7d6e09850ae1c749e88675db80c1eb)